src/docker-compose-environment.ts

```diff
@@ -109,7 +109,12 @@ export class DockerComposeEnvironment {
 
   private async findStartedContainers(dockerClient: DockerClient): Promise<Dockerode.ContainerInfo[]> {
     const containers = await dockerClient.listContainers();
-    return containers.filter((container) => container.Labels["com.docker.compose.version"] !== undefined);
+    return containers.filter(
+      (container) =>
+        container.Labels["com.docker.compose.version"] !== undefined &&
```

Comment:
This is great, I hadn't noticed this. If we can filter by the workdir dir and config file we probably don't need com.docker.compose.version, I had added it just to get all containers started by docker-compose

Reply:
sorry, was going to do this as soon as I got home.